x/incentives/client/cli/query.go

```diff
@@ -384,31 +386,80 @@ $ %s query incentives rewards-estimation
 				return err
 			}
 
-			lockIdStrs := strings.Split(lockIdsCombined, ",")
-			lockIds := []uint64{}
-			for _, lockIdStr := range lockIdStrs {
-				lockId, err := strconv.ParseUint(lockIdStr, 10, 64)
+			var res *lockuptypes.AccountLockedLongerDurationResponse
+			if owner != "" {
+				queryClientLockup := lockuptypes.NewQueryClient(clientCtx)
+
+				res, err = queryClientLockup.AccountLockedLongerDuration(cmd.Context(), &lockuptypes.AccountLockedLongerDurationRequest{Owner: owner, Duration: time.Millisecond})
 				if err != nil {
 					return err
 				}
-				lockIds = append(lockIds, lockId)
+			} else {
+				owner = ""
+			}
```

Comment:
Why would we do this? So if the error is not nil, we set the owner to an empty string?

Reply:
I wasn't sure, if an owner flag is not provided, does it still initialize an empty owner string? The below logic requires this so thats why I was explicit with it.

Reply:
Ahh I see, I misread this and thought that this applies to:
```
if err != nil {
    return err
}
```

Anyways, a string is "" by default so we can remove this code branch